pyface/tasks/task_layout.py

```diff
@@ -73,9 +73,9 @@ class LayoutContainer(LayoutItem):
 
     items = List(pretty_skip=True)
 
-    def __init__(self, *items, **traits):
-        super(LayoutContainer, self).__init__(**traits)
-        self.items = list(items)
+    def __init__(self, *args, **kwargs):
```

Comment:
Leave `**kwargs` as `**traits` as this is common usage for keyword args of traits `__init__` methods.